src/cpp/statistics/rtps/monitor-service/MonitorService.cpp

```diff
@@ -287,62 +262,28 @@ bool MonitorService::write_status(
                 status_data.local_entity(to_statistics_type({local_participant_guid_.guidPrefix, entity_id}));
                 GUID_t local_entity_guid = {local_participant_guid_.guidPrefix, entity_id};
```

Comment:
I am getting a warning building locally:

```
warning: variable ‘local_entity_guid’ set but not used [-Wunused-but-set-variable]
  263 |                 GUID_t local_entity_guid = {local_participant_guid_.guidPrefix, entity_id};
```

The variable is used but not in every case. In order to remove the warning, please use the variable casting it to void.